code/datums/wires/wires.dm

```diff
@@ -248,14 +248,16 @@
 	switch(action)
 		if("cut")
 			if(istype(I, /obj/item/wirecutters) || IsAdminGhost(usr))
-				playsound(holder, I.usesound, 20, 1)
+				if(I)

```

Comment:
no need for this check

Reply:
This is not `attackby`, it's `topic`. So there is a need for this check, because `AdminGhost` is highly unlikely to have an `/obj/item` in hand.